src/Microsoft.DotNet.Interactive.SqlServer/ToolsServiceKernel.cs

```diff
@@ -374,7 +374,7 @@ Task IKernelCommandHandler<RequestValueInfos>.HandleAsync(RequestValueInfos comm
         var valueInfos = QueryResults.Keys.Select(key =>
         {
             var formattedValues = FormattedValue.CreateSingleFromObject(
-                _variables[key],
+                QueryResults[key],
```

Comment:
Do we need to be storing these in separate buckets?

Reply:
changed my mind : yes should be one, merged that behaviour